susemanager-sync-data/oval.config.json

```diff
@@ -0,0 +1,144 @@
+{
+  "sources": {
```

Comment:
At the moment, patch information is sourced from channels. Eventually, we'll start using OVAL patch info to gather additional metadata not available through channels, such as patch status (e.g., in QA or released). The patch info source was added to the JSON file, to prevent compatibility problems in the future.